optional_plugins/varianter_cit/avocado_varianter_cit/__init__.py

```diff
@@ -0,0 +1,197 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+
+import hashlib
+import itertools
+import os
+import sys
+
+from six import iteritems
+from six.moves import configparser
+
+from avocado.core import exit_codes
+from avocado.core.output import LOG_UI
+from avocado.core.plugin_interfaces import CLI
+from avocado.core.plugin_interfaces import Varianter
+from avocado.core.tree import TreeNode
+
+
+class VarianterCitCLI(CLI):
+
+    """
+    CIT Varianter options
+    """
+
+    name = 'cit'
+    description = "CIT Varianter options for the 'run' subcommand"
+
+    def configure(self, parser):
+
+        for name in ("run", "variants"):  # intentionally ommiting "multiplex"
```

Comment:
It's OK to even remove the comment here about the multiplexer.

Reply:
ok